README.md

```diff
@@ -40,4 +40,4 @@ See the LICENSE file.
 
 ## Contributing
 
-See the [CONTRIBUTING](doc/contributing.md) file.
\ No newline at end of file
+See the [CONTRIBUTING](doc/contributing.md) file.
```

Comment:
I think the end of line here is incorrect, that's why you have a diff without a change.
I also think that it makes conflicts the PR. Could you please fix this and the PR will be applied very fast then?

Thanks a lot.
Regards,